images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
+      com.redhat.component="$NAME" \
+      name="$FGC/$NAME" \
+      version="$VERSION" \
+      usage="This image is meant to be used with the toolbox command" \
+      summary="Base image for creating Ubuntu 19.04 toolbox containers" \
+      maintainer="Debarshi Ray <rishi@fedoraproject.org>"
+
+COPY README.md /
+
+RUN apt-get update
+
+RUN apt-get -y upgrade
+
+COPY extra-packages /
+RUN apt-get -y install $(cat extra-packages | xargs)
+RUN rm /extra-packages
+
+RUN sed -i -e 's/ ALL$/ NOPASSWD:ALL/' /etc/sudoers
+
+# Restore documentation
+RUN yes | /usr/local/sbin/unminimize
+RUN apt-get clean
+
+RUN echo VARIANT_ID=container >> /etc/os-release
```

Comment:
Interesting. Do the Ubuntu base images have any `VARIANT_ID`? If so, what's it?

Reply:
It's not part of the base image, and toolbox barfs if it's not there.

Reply:
I think toolbox is wrong here, VARIANT_ID is an optional field.

Reply:
Yes, the `VARIANT_ID` shouldn't be critical - the Toolbox logic doesn't need it to be.

Feel free to file a separate PR to fix the code to not barf.

Reply:
I'll experiment a bit with omitting that one, but it'll be a PR of its own.